src/mesh/mesh.jl

```diff
@@ -326,24 +326,46 @@ function _build_faces!(c2n::MeshConnectivity, celltypes)
     return _face_types, c2f, f2c, f2n
 end
 
+"""
+    _build_boundary_faces!(f2n::MeshConnectivity, f2c::MeshConnectivity, bc_nodes)
+
+Find boundary faces (all faces with only 1 adjacent cell) from the face-to-node,
+face-to-cell connectivities, and the list of tagged nodes (i.e, list of the nodes
+belonging to each bc).
+"""
 function _build_boundary_faces!(f2n::MeshConnectivity, f2c::MeshConnectivity, bc_nodes)
     @assert (bc_nodes ≠ nothing) "bc_names and bc_nodes must be defined"
     @assert length(indices(f2n)) == length(indices(f2c)) "invalid f2n and/or f2c"
     @assert length(indices(f2n)) ≠ 0 "invalid f2n and/or f2c"
 
-    # find boundary faces (all faces with only 1 adjacent cell)
-
+    # Allocate
     f2bc_numindices = zeros(Int, length(indices(f2n)))
     f2bc_indices = Vector{Int}()
     sizehint!(f2bc_indices, length(f2bc_numindices))
 
+    # To ease nodes identification in `bc_nodes`, create,
+    # for each bc, a vector of size `nnodes(mesh)` who tells
+    # if each nodes belongs to the concerned BC
+    # Rq 1: since we don't have access to `nnodes(mesh)` here, we compute
+    # using the functions arguments
+    # Rq 2: for element of order > 1, there might be "inside nodes", not included in "f2n",
```

Comment:
`f2n` is built from `f2n_from_c2n`  (in [_build_faces!](https://github.com/bcube-project/Bcube.jl/blob/5e7f840b278a2e82aa6a2f4845e8aa354cac9c57/src/mesh/mesh.jl#L293C36-L293C48)), and this function calls `faces2nodes` that returns "inside nodes" as far as I understand

Reply:
Live discussion : I was referring to nodes inside the volume, not inside the face